### DISCUSSION TITLE
Option to automatically play the next file in the folder.

### DISCUSSION THREAD

**Discussion by Nucleorion** [smplayer-dev/smplayer#315] [Ideas]
Greetings,

If you doubt it is the best player, the fastest to jump forward, and the only one that does frame by frame backwards in a fluid way. But it is not my default player to watch series and videos since you have to make playlists. We already have the list in the folder since the files are called 1x01 1x02 and so on. So my suggestion is that when you finish playing a file there is the option to continue playing the next file in the folder in name order.

**Comment by smplayer-dev** [comment]
Preferences -> Playlist, enable "add files to the playlist automatically", set "add files from folder" to "video files" (or "consecutive files"). Enable also "play next file automatically".

**Comment by Nucleorion** [comment]
It does not have the desired effect. Just play the next file by name. Do not add anything to the playlist. Just add to a window where you can see the history.
If I do what you say and open the 1x01 file when it ends or when I click next, it does not play the 1x02.
Imagine that I am watching the chapter 1x03 Series A.mkv and then I want to see the chapter 2x01 Series B.mkv The next day I want to continue watching the series A, if there was a window with the history I would only have to click on the chapter in the window and continue playback. As at the end it will automatically load the file 1x04 Serie A.mkv, I will be able to see without having to go around creating playlists several series simultaneously.
